biocypher_metta/adapters/cell_line_ontology_adapter.py

```diff
@@ -5,8 +5,8 @@ class CellLineOntologyAdapter(OntologyAdapter):
         'clo': 'http://purl.obolibrary.org/obo/clo.owl'
     }
     
-    def __init__(self, write_properties, add_provenance, ontology, type, label='clo', dry_run=False):
-        super(CellLineOntologyAdapter, self).__init__(write_properties, add_provenance, ontology, type, label, dry_run)
+    def __init__(self, write_properties, add_provenance, ontology, type, label='clo', dry_run=False, add_description=False):
```

Comment:
In other adapters like `gene_ontology_adapter`, `add_description` is `True` by default, and here it's `False`. There should be consistency and the option should be set to `False` by default.